app/views/finance/participants/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: finance_participants_path) %>
+
+<h1 class="govuk-heading-xl">Participant</h1>
+
+<h2>Identities</h2>
+
+<% @user.participant_identities.each_with_index do |identity, i| %>
+  <h3><%= "Identity #{i + 1}" %></h3>
+
+  <dl class="govuk-summary-list">
```

Comment:
❤️  `dl`/`dt`

Reply:
whoops forgot to change this one to `govuk_summary_list` which does that under the hood